apollo-router/Cargo.toml

```diff
@@ -97,6 +97,7 @@ humantime = "2.1.0"
 humantime-serde = "1.1.1"
 hyper = { version = "0.14.27", features = ["server", "client"] }
 hyper-rustls = { version = "0.24.1", features = ["http1", "http2"] }
+hyper-trust-dns-connector = "0.4.0"
```

Comment:
that crate's last release is over a year old now and it brings in old versions of trust-dns. Considering the amount of code this represent, we could have an in house implementation: https://github.com/paullgdc/hyper-trust-dns-connector/blob/master/src/lib.rs

Reply:
Yep. I did have those thoughts. Ok. I'll do an in-house one since we think it's useful to have.

Reply:
done: ef2dfcdfa9c31b9b9d07785e330be7c391ed61d0